shell/common/snapshot_controller_impeller.cc

```diff
@@ -12,6 +12,9 @@
 #include "flutter/impeller/display_list/dl_image_impeller.h"
 #include "flutter/impeller/geometry/size.h"
 #include "flutter/shell/common/snapshot_controller.h"
+#include "impeller/renderer/render_target.h"
+
+#define EXPERIMENTAL_CANVAS false
```

Comment:
Is there some context on why this isn't in a GN rule somewhere and applied globally instead of on a per TU basis?

Reply:
factoring out to a gn rule seems like a good idea, will make it easier to flip on/off to see if there are any golden diffs or test failures.

Reply:
Don't mean to block this patch though. We can do that later.

Reply:
It was easy, done.